crossbundle/cli/src/commands/build/build_context.rs

```diff
@@ -39,9 +35,11 @@ impl BuildContext {
             .custom_metadata()
             .ok_or(Error::InvalidManifestMetadata)?
             .to_owned();
+        println!("custom_metadataa is {:?}", custom_metadata);
         let metadata = custom_metadata
             .try_into::<Metadata>()
             .map_err(|_| Error::InvalidManifestMetadata)?;
+        println!("metadata is {:?}", metadata);
```

Comment:
Remove printlns

Reply:
Not All features I wanted to implement were done. I still need it to debug.